src/api/goods.ts

```diff
@@ -5,15 +5,26 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => response.json())
+    .catch(error => {
+      throw new Error(error);
+    });
 }
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods
+      .sort((currGood, nextGood) => currGood.name.localeCompare(nextGood.name))
+      .slice(0, 5))
+    .catch(error => {
+      throw new Error(error);
+    });
```

Comment:
if you throw error this simple, you don't need to do it every time, you already handle it in `getAll`
otherwise, if you specify error kind (like if it occur here or there), catching it here and there makes sense
```suggestion
    .then(goods => (
      [...goods]
        .sort((a, b) => a.name.localeCompare(b.name))
        .slice(0, 5)
    ));
```